stargate-starter/src/main/java/io/stargate/starter/Starter.java

```diff
@@ -307,6 +308,14 @@ public Starter(
     this.disableMBeanRegistration = true;
   }
 
+  private static void log(String message, Level level) {
+    System.out.println("[" + level.getName() + "] " + message);
+  }
+
+  private static void logerr(String message, Level level) {
+    System.err.println("[" + level.getName() + "] " + message);
```

Comment:
Should we consider passing the throwable exception 'e' here and print the stack trace like below, instead of just the message? 
`e.printStackTrace(System.err);`
Relying just on the message (and when that being some generic message) might sometimes make it hard to find which part of the code caused the exception and the stack trace will give a quick idea with the line number.

Reply:
Usually I worry about verbosity, but here I agree that stack trace would make lots of sense. These should be rare and real failures.

Reply:
sure, and these are things that prevent startup